app/models/user.js

```diff
@@ -40,5 +43,9 @@ export default Addressable.extend({
 
   fullName: function(){
     return (this.get('firstName') + " " + this.get('lastName'));
-  }.property('firstName', 'lastName')
+  }.property('firstName', 'lastName'),
+
+  lName: function(){
```

Comment:
I've made a new "system" permission now. First name: GoodCity; Last name: Team. And removed the lName function below. https://github.com/crossroads/api.goodcity/pull/59